inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java

```diff
@@ -289,7 +292,13 @@ private void invokeDeleteProcess(String groupId, String operator) {
         InlongGroupInfo groupInfo = groupService.doDeleteCheck(groupId, operator);
         // start to delete group process
         GroupResourceProcessForm form = genGroupResourceProcessForm(groupInfo, GroupOperateType.DELETE);
-        workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
+        WorkflowResult result = workflowService.start(ProcessName.DELETE_GROUP_PROCESS, operator, form);
+        List<TaskResponse> tasks = result.getNewTasks();
+        if (TaskStatus.FAILED.equals(tasks.get(tasks.size() - 1).getStatus())) {
+            String errMsg = String.format("failed to delete group process for groupId=%s", groupId);
+            LOGGER.error(errMsg);
```

Comment:
It is better that
LOGGER.error("failed to delete group process for groupId={}", groupId);

when log level is fatal, errMsg object will not be created.

Reply:
1. We are all at least at the `info` level. No matter in the test or production environment, we use logs at the `info` level at least;
2. The following `WorkflowListenerException` also uses this error message, so it reuses the error message.